lua/ulx/modules/sh/proximity_voice_commands.lua

```diff
@@ -10,5 +10,20 @@ end
 local proximityGlobal = ulx.command( CATEGORY_NAME, "ulx enableproximityvoice", forceProximityVoice )
 proximityGlobal:addParam{ type=ULib.cmds.BoolArg, invisible=true }
 proximityGlobal:defaultAccess( ULib.ACCESS_ADMIN )
-proximityGlobal:help( "Enabled proximity voice for everyone" )
+proximityGlobal:help( "Enable proximity voice for everyone" )
 proximityGlobal:setOpposite( "ulx disableproximityvoice", {nil, true} )
+
+
+local function proximityVoiceOverride( caller, targets, enabled )
+    for _, ply in pairs( targets ) do
+        ProximityVoiceOverridePlayerConfig( ply, enabled )
+    end
+
+    local word = enabled and "disabled" or "enabled"
```

Comment:
```suggestion
    local word = enabled and "enabled" or "disabled"
```?

Reply:
The way I have it is correct
![image](https://user-images.githubusercontent.com/11771225/130156559-eac45ac6-1070-4e31-abbf-e062581053ed.png)

Reply:
yeah forgot to apply my changes from gcompute